### ISSUE TITLE
Doesn't show child classes

### ISSUE THREAD

**Issue by lance-shi** [afawcett/apex-umlcanvas#17]
The tool doesn't show the relationship for child classes. Some child classes are declared as public and referenced everywhere. But this tool is not showing them

**Comment by JohnDTheMaven** [comment]
@lance-shi - Which version of the apex-toolingapi are you using?  SOAP or REST?

**Comment by lance-shi** [comment]
I am using version 1.5. Should be SOAP

**Comment by afawcett** [comment]
The reason for this is, last i checked, which addmitly was year back, the Symbol Table info for the Tooling API didn't qualify inner classes e.g. OuterClass.InnerClass, e.g. only gave InnerClass back, which makes it useless to distinguish between top level or child classes when building the class list.

**Comment by rickroesler** [comment]
I ran across this repo a couple of weeks ago. As noted in this issue, the current release does not support inner classes. In reviewing the SymbolTable documentation, it looks like this information is available. 

My big question: is this project still of interest and actively being worked on? if so, I'd like to branch and work on the inner class functionality.

Please advise.

**Comment by afawcett** [comment]
@rickroesler  yes and yes please! I continue to see interest in this tool and thus put some work into it a few months ago to fix some issues. Thank you!

**Comment by rickroesler** [comment]
@afawcett Roger. Things have gotten very busy at work. I'll let you know as I make progress.

**Comment by stoevie** [comment]
> I ran across this repo a couple of weeks ago. As noted in this issue, the current release does not support inner classes. In reviewing the SymbolTable documentation, it looks like this information is available.
> 
> My big question: is this project still of interest and actively being worked on? if so, I'd like to branch and work on the inner class functionality.
> 
> Please advise.

Yes Rick, I am also interested in the support of inner classes.

**Comment by rickroesler** [comment]
> @afawcett Roger. Things have gotten very busy at work. I'll let you know as I make progress.

Started looking at this yesterday. It looks like all the data is there to render inner class information and create <<use>> links. 

FYI - the ExternalReference array item stores the inner class name as 'name' and its surrounding outer class as 'namespace'.
